client/my-sites/subscribers/hooks/use-add-subscribers-callback.ts

```diff
@@ -15,12 +15,11 @@ import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
 export const useAddSubscribersCallback = ( siteId: number | null ) => {
 	const { completedJob } = useActiveJobRecognition( siteId ?? 0 );
 	const importError = useImportError();
-	const { isJetpackNonAtomic, selectedSiteSlug } = useSelector( ( state ) => {
-		return {
-			isJetpackNonAtomic: isJetpackSite( state, siteId, { treatAtomicAsJetpackSite: false } ),
-			selectedSiteSlug: getSelectedSiteSlug( state ),
-		};
-	} );
+	const isJetpackNonAtomic = useSelector( ( state ) =>
+		isJetpackSite( state, siteId, { treatAtomicAsJetpackSite: false } )
+	);
+	const selectedSiteSlug = useSelector( getSelectedSiteSlug );
```

Comment:
This is another place where we shouldn't look at the selected site. There's a `siteId` param, let's use that! Fixing in #103495.